solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -135,9 +135,10 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
             _amount
         );
 
-        // return the amount minus the conversion fee, the conversion fee.
+        /// @dev fee is the total fee

```

Comment:
in the sense that it is the total of conversion fee amount + protocol fee amount, right ?
Is ambiguous because we also have a total of fees held in the contract.

Reply:
No, the `fee` here refers to the conversion fee, and the conversion fee is the total fee that will be charged to the user.
And protocol fee is just part of the conversion fee.